_pages/tools/slack/integrations.md

```diff
@@ -9,7 +9,7 @@ questions:
 We have a few different Slack Apps and bots you'll see in Slack:
 
 - [Charlie](https://app.slack.com/client/T025AQGAN/D01NBG7T3PF): our [Slack app](https://api.slack.com/). Knows all kinds of tricks! Check out [its repo on GitHub](https://github.com/18F/charlie) to learn more.
-- [coffeemate](https://gsa-tts.slack.com/team/charlie): send a message with `coffee me` in a public channel or direct message [@Charlie](https://app.slack.com/client/T025AQGAN/D01NBG7T3PF) with `coffee me` to set up a virtual coffee/tea with a random coworker.
+- [coffeemate](https://app.slack.com/client/T025AQGAN/D01NBG7T3PF): send a message with `coffee me` in a public channel or direct message [@Charlie](https://app.slack.com/client/T025AQGAN/D01NBG7T3PF) with `coffee me` to set up a virtual coffee/tea with a random coworker.
```

Comment:
Hmm, are you sure these links are correct? I land in "#unknown-channel".

Reply:
You are correct. Those are links to my DMs with Charlie, so nobody else would have access. I'll see if I can figure out how to link to the bot in Slack, but it seems like it might not be possible anymore since Charlie is no longer an actual user.

Reply:
Maybe these will work